src/styles/blocks/_container.scss

```
@@ -0,0 +1,23 @@
+
+.container {
+  display: grid;
+  place-content: center;
+  grid-template-columns:repeat(1, $card-width);
+  gap: $horyzontal-gap $vertical-gap;
+  padding: $horizontal-padding $vertical-padding;
+
+@media screen and (min-width: $small-screen) {
+  grid-template-columns: repeat(2, $card-width);
+  place-self: center;
+}
+
+@media (min-width: $medium-screen) {
+  grid-template-columns: repeat(3, $card-width);
+  place-self: center;
+}
+
+@media (min-width: $large-screen) {
+  grid-template-columns: repeat(3, $card-width);
+  place-self: center;
+}
```

use prettier to format your code 
```suggestion
    @media screen and (min-width: $small-screen) {
      grid-template-columns: repeat(2, $card-width);
      place-self: center;
    }
    
    @media (min-width: $medium-screen) {
      grid-template-columns: repeat(3, $card-width);
      place-self: center;
    }
    
    @media (min-width: $large-screen) {
      grid-template-columns: repeat(3, $card-width);
      place-self: center;
    }
}
```